documentation/docs/steps/dockerExecute.md

```diff
@@ -35,6 +31,28 @@ dockerExecute(dockerImage: 'maven:3.5-jdk-7'){
 }
 ```
 
+## Kubernetes support
+If the Jenkins is setup on a Kubernetes cluster, then you can execute the closure inside a container of pod by setting an environment variable `ON_K8S` to `true`. However, it will ignore both `dockeOptions` and `dockerVolumeBind` values.
```

Comment:
Maybe one comment on that ON_K8S should be set not in the pipeline but globally as env variable

Reply:
What about this comment?

Reply:
Documentations are yet to be modified. I will change this in next commit